lib/MusicBrainz/Server/Controller/Instrument.pm

```diff
@@ -157,10 +157,12 @@ sub list : Path('/instruments') Args(0) {
     }
 
     $c->stash(
-        instruments_by_type => $entities,
-        instrument_types => \@types,
         current_view => 'Node',
         component_path => 'instrument/List',
+        component_props => {
```

Comment:
It's a bit unsatisfactory to create broken commits like this - if someone bisecting lands on this commit, the page will be broken for them because the component isn't changed to understand this until b137a99a2a11c486bd1050f9c8f2f9a45925760e from what I can tell.

Reply:
Fixed.